tests/property/ic_api/arg_data_raw/src/index.ts

```diff
@@ -0,0 +1,61 @@
+import {
+    argDataRaw,
+    candidEncode,
+    IDL,
+    init,
+    postUpgrade,
+    query,
+    update
+} from 'azle';
+
+export default class {
+    initArgDataRaw: Uint8Array | null = null;
+    postUpgradeArgDataRaw: Uint8Array | null = null;
+
+    @init([IDL.Text])
+    init(arg: string): void {
+        console.log('init', arg);
```

Comment:
Make sure all are console.info

Reply:
Done. I made them console.infos because I want to have the parameters and not have there be an error about not using them

Reply:
switched to underscore